app/src/cobrand/java/cm/aptoide/pt/billing/view/login/PaymentLoginFlavorPresenter.java

```diff
@@ -0,0 +1,142 @@
+package cm.aptoide.pt.billing.view.login;
```

Comment:
I don't really like to use inheritance here for two reasons:

1. Conceptually there's no real reason for inheritance. The only advantage here is maybe to avoid duplicated code? Also, if we have X flavours, we will need X implementations of these presenters. Unlikely to be a problem but still a possible scalability issue.

2. It ironically leads to duplicated code. Check 'handleAptoideSignupEvent()', for example. The only difference between flavours is two lines and you need to implement the full method to even have signup. If we were to create a new flavour, we would need to add this method to have sign up. Not very good.

I do think composition is better here. Either have a 'PolicyManager' where it would hold whether we should show PP and TC or not, and other related behaviour. Consequently only have one presenter where it would act accordingly if we should or not show. This has the benefit of minimising repeated code, and also being able to hide the TC/PP on the vanilla flavour if there's any reason in the future.